File: VB/CustomGridWinApp1/Form1.vb
Imports System
Imports System.Windows.Forms

Namespace CustomGridWinApp1
	Partial Public Class Form1
		Inherits Form

		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
			Me.ordersTableAdapter.Fill(Me.nwindDataSet.Orders)
			Me.employeesTableAdapter.Fill(Me.nwindDataSet.Employees)
		End Sub
	End Class
End Namespace

File: CS/CustomGridWinApp1/Form1.cs
using System;
using System.Windows.Forms;

namespace CustomGridWinApp1
{
	public partial class Form1 : Form
	{
		public Form1()
		{
			InitializeComponent();
		}

		private void Form1_Load(object sender, EventArgs e)
		{
			this.ordersTableAdapter.Fill(this.nwindDataSet.Orders);
			this.employeesTableAdapter.Fill(this.nwindDataSet.Employees);
		}
	}
}
